src/format.rs

```diff
@@ -0,0 +1,366 @@
+use super::utils::StrCursor;
+use crate::FormatError;
+
+use core::fmt;
+
+use spinoso_time::tzrs::Time;
```

Comment:
to avoid the dependency on `spinoso-time` and to allow this `strftime` parser to be used elsewhere in the ecosystem, I was thinking this crate might define a trait that defines all of the accessors needed for this parser. What do you think?

Reply:
Yes I think this would be the best solution.